debian/control

```diff
@@ -136,7 +136,7 @@ Architecture: i386 amd64 armhf arm64
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${misc:Depends},
-         mir-platform-graphics-android15,
+         mir-platform-graphics-android-caf15,
          mir-client-platform-android5,
```

Comment:
this also needs to depend on cat :)

Reply:
On cat? ^^

Reply:
> On cat? ^^

i mean caf :P

Reply:
And how to do that? ^^
Should I add android-caf to the dependency?

Reply:
Replace `mir-client-platform-android5`  with `mir-client-platform-android-caf5`